django_rq/views.py

```diff
@@ -51,6 +52,11 @@ def stats(request):
             queue_data['finished_jobs'] = len(finished_job_registry)
             queue_data['started_jobs'] = len(started_job_registry)
             queue_data['deferred_jobs'] = len(deferred_job_registry)
+            try:
+                scheduler = get_scheduler(queue.name)
+                queue_data['scheduled_jobs'] = len(scheduler.get_jobs())
```

Comment:
I started to implement a `scheduler.count()` method using Redis `ZCOUNT`, but if you have multiple RQ queues defined in Django (in the same Redis instance), `ZCOUNT` will just return all jobs scheduled for all queues, as `scheduler.get_jobs()` does now when it retrieves all jobs.

The `origin` of the `Job` has the name of the queue the job will be eventually queued on, but I can't think how to filter for that at the Redis level without retrieving all jobs. I can't think of an efficient way to get the count of jobs scheduled for each queue without retrieving all jobs and manually filtering.

I have made an issue at https://github.com/ui/rq-scheduler/issues/118 and will make the PR for `count()` anyway, perhaps with an optional queue name.